tensorflow-app-starters-test-support/src/main/java/org/springframework/cloud/stream/app/test/tensorflow/TensorflowProcessorTestConfiguration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 the original author or authors.
```

Comment:
Today is `2017`, not `2016` any more.
So, Copyrights have to reflect the changes for the current year.
You should adjust your IDE to automatically do that.

Not a big deal though: I'll fix on merge.